CONTRIBUTING.md

```diff
@@ -69,22 +69,21 @@ charmcraft pack
 ### Deploy
 
 OpenSearch has a set of system requirements to correctly function, you can find the list [here](https://opensearch.org/docs/2.6/opensearch/install/important-settings/).
-To set those settings using cloudinit-userdata:
+Some of those settings must be set using cloudinit-userdata on the model, while others must be set on the host machine:
 ```bash
-# Create a cloudinit-userdata file, to set the required system settings of opensearch.
 cat <<EOF > cloudinit-userdata.yaml
 cloudinit-userdata: |
   postruncmd:
-    - [ 'sysctl', '-w', 'vm.max_map_count=262144' ]
-    - [ 'sysctl', '-w', 'vm.swappiness=0' ]
-    - [ 'sysctl', '-w', 'net.ipv4.tcp_retries2=5' ]
-    - [ 'sysctl', '-w', 'fs.file-max=1048576' ]
+    - [ 'echo', 'vm.max_map_count=262144', '>>', '/etc/sysctl.conf' ]
+    - [ 'echo', 'vm.swappiness=0', '>>', '/etc/sysctl.conf' ]
+    - [ 'echo', 'net.ipv4.tcp_retries2=5', '>>', '/etc/sysctl.conf' ]
+    - [ 'echo', 'fs.file-max=1048576', '>>', '/etc/sysctl.conf' ]
+    - [ 'sysctl', '-p' ]
 EOF
-```
 
-or in a single machine:
-```bash
-sudo sysctl -w vm.max_map_count=262144 vm.swappiness=0 net.ipv4.tcp_retries2=5
+echo "vm.max_map_count=262144" | sudo tee -a /etc/sysctl.conf
```

Comment:
Missing: `net.ipv4.tcp_retries2=5`

Reply:
And missing: `fs.file-max

Reply:
Those are applied in the LXD container directly through cloudinit `postruncmd`. Other kernel properties like `vm.swappiness` must be applied to the host